py4syn/epics/QE65000Class.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, mnemonic, pv=None, responseTimeout=15, output="./out",
         self.pvDarkCorrection = PV(pv+":ElectricalDark")
 
         # the spectra come from different pv if use darkcorrection
-        if self.pvDarkCorrection.get() == "ON":
+        if self.pvDarkCorrection.get() == 1:
```

Comment:
@gabrielfedel, is it possible to change during runtime or it is configured at the IOC and available as read-only on this PV?
If it is possible to change during runtime I would recommend to have some sort check on the moment that you read the pvSpectrum to point to the correct value.